src/app/Caches/Create/ConfigurationBasic.tsx

```diff
@@ -116,51 +129,43 @@ const ConfigurationBasic = ({ formConfigurationBasic, handleFormConfigurationBas
         );
     };
 
-    // Options for Number Owners
-    const optionsNumberOwners = [
-        <SelectOption key={0} value="Select" isPlaceholder />,
-        <SelectOption key={1} value="1" />,
-        <SelectOption key={2} value="2" />,
-        <SelectOption key={3} value="3" />,
-        <SelectOption key={4} value="4" />,
-        <SelectOption key={5} value="5" />,
-        <SelectOption key={6} value="6" />,
-        <SelectOption key={7} value="7" />,
-        <SelectOption key={8} value="8" />,
-        <SelectOption key={9} value="9" />,
-        <SelectOption key={10} value="10" />,
-    ];
-
     // Form Number of Owners
     const formNumberOwners = () => {
         return (
             <React.Fragment>
                 <FormGroup
-                    label="Number of owners"
                     fieldId='field-number-owners'
-                    isRequired={topology === 'distributed'}> {/*Required when topology is distributed */}
-                    <Tooltip content="Tooltip for number of owners"><OutlinedQuestionCircleIcon /></Tooltip>
-                    <Select
-                        variant={SelectVariant.single}
-                        aria-label="Select Input"
-                        onToggle={onToggleNumberOwners}
-                        onSelect={onSelectNumberOwners}
-                        selections={selectedNumberOwners}
-                        isOpen={isOpenNumberOwners}
-                        aria-labelledby="toggle-id-number-owners"
-                    >
-                        {optionsNumberOwners}
-                    </Select>
+                    isRequired={topology as CacheType == CacheType.Distributed}> {/*Required when topology is distributed */}
+                    <MoreInfoTooltip label={t('caches.create.configurations.basic.number-owners')} toolTip={t('caches.create.configurations.basic.number-owners-tooltip')} textComponent={TextVariants.h4} />
+                    <NumberInput
+                        value={selectedNumberOwners}
+                        min={minValue}
+                        max={maxValue}
+                        onMinus={onMinus}
+                        onChange={onChange}
+                        onPlus={onPlus}
+                        inputName="input"
+                        inputAriaLabel="number input"
+                        minusBtnAriaLabel="minus"
+                        plusBtnAriaLabel="plus"
+                        widthChars={2}
+                    />
                 </FormGroup>
-            </React.Fragment>
+            </React.Fragment >
         )
     }
 
     // Options for Encoding Cache
     const optionsEncodingCache = [
         <SelectOption key={0} value="JSON" isPlaceholder />,
-        <SelectOption key={1} value="YAML" />,
-        <SelectOption key={2} value="XML" />,
+        <SelectOption key={1} value="Java" />,
```

Comment:
@karesti value here is mentioned as a string, instead of EncodingType ref from InfinispanRefData.ts, Is this the right way? not sure!

Reply:
you can display the EncodigType as string